dev-2/Yandex/Yandex/Pages/AuthorizationPage.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Yandex.Utils;
+using OpenQA.Selenium;
+
+namespace Yandex.Pages
+{
+    public class AuthorizationPage : BasePage
+    {
+        private AuthorizationLocators authorizationLocators;
+            
+        public AuthorizationPage(IWebDriver driver) : base(driver)
+        {
+            authorizationLocators = new AuthorizationLocators();
+        }
+
+        protected override string Url => "";
+
+        public void LoginAuthorizationPage()
+        {
+            DataInput(authorizationLocators.userName, authorizationLocators.userNameInput);
+            NavigateOnButton(authorizationLocators.loginButtonForAccount);
+            DataInput(authorizationLocators.password, authorizationLocators.passwordInput);
+        }
+
+        public bool isLoLoginAuthorizationPage()
+        {
+            LoginAuthorizationPage();
+
+            Wait.Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementExists(authorizationLocators.messageStatusField));
+            var messageStatus = Driver.FindElement(authorizationLocators.messageStatusField).Text;
+
+            return messageStatus != null;
```

Comment:
If you would like to check your text value I recommend you to use this method here - string.IsNullOrEmpty()